arvo/eyre/examples.md

```diff
@@ -526,9 +521,9 @@ The sample of the second nested gate must be:
 [authenticated=? =request:http]
 ```
 
-The return type of the generator must be [simple-payload:http](@/docs/arvo/eyre/data-types.md#simple-payload-http).
+The return type of the generator must be [simple-payload:http](@/docs/arvo/eyre/data-types.md#simple-payload-http). If you look at our example generator you'll see it meets these requirements.
 
-If you look at our example generator you'll see it meets these requirements.
+Because generators return the entire HTTP message as a single `simple-payload`, Eyre can calculate the `content-length` itself and automatically add the header, so we don't need to do it by hand like with a Gall agent.
```

Comment:
fwiw, I don't think you _need_ to specify the `content-length` header explicitly anywhere. If you don't include it, the runtime will send with `transfer-encoding: chunked` instead. Not sure if this is worth including here though.